app/src/ui/toolbar/push-pull-button.tsx

```diff
@@ -178,7 +178,7 @@ export class PushPullButton extends React.Component<IPushPullButtonProps, {}> {
     }
 
     if (tipState === TipState.Unborn) {
-      return 'Cannot publish unborn repository'
+      return 'Cannot publish unborn HEAD'
```

Comment:
Can we say "branch" instead of "HEAD" here? Does that fit without truncating?

Reply:
That might cause confusion since the branch that is being pushed is available but the initial commit of the HEAD was reverted, thus making the HEAD unborn.

Reply:
Probably easier for users to google "unborn HEAD" too 👍